go.mod

```diff
@@ -120,7 +119,7 @@ require (
 	github.com/shirou/gopsutil v0.0.0-20190901111213-e4ec7b275ada
 	github.com/stretchr/testify v1.4.0
 	github.com/syndtr/goleveldb v1.0.0
-	github.com/tron-us/go-btfs-common v0.0.10
+	github.com/tron-us/go-btfs-common v0.1.3
```

Comment:
here you may need double check, the v0.1.3 is too old

Reply:
sure, will do